Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -93,6 +95,24 @@ defineSuite([
     var uberLabelColor = Color.fromBytes(0xee, 0xee, 0xee, 0xee);
     var uberLabelScale = 4;
 
+    var oldConsoleLog;
+    var consoleMessages;
+
+    beforeAll(function() {
+        oldConsoleLog = console.log;
+        console.log = function(msg) {
```

Comment:
We should use spies for this instead of just replacing `console.log`.  We should also call through to the underlying `console.log` so that the real call happens.